core/Azure.Mcp.Core/src/Areas/Server/Options/TransportType.cs

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Text.Json.Serialization;
```

Comment:
If this is staying as a `class` with `const string` fields, then you probably don't need to add this `using` statement.

Reply:
yes, good catch, removed it.